buildscripts/ci/release/make_tag_name.sh

```diff
@@ -39,8 +39,6 @@ function version_exists()
 {
     local version="$1" label="${2-}" # label is optional
     local suffix="${label:+-${label}}" # prepend hyphen if label not empty
-    version="${version%.0}" # strip patch number if it's zero
-    version="${version%.0}" # strip minor number if it's zero
```

Comment:
These two lines should be retained. This is checking for existing tags rather than creating a new one.

For example, if a tag exists called `v5` or `v5.0` then these lines will prevent us creating a new tag called `v5.0.0`.

Without these lines, only an existing tag called `v5.0.0` would prevent a new tag with this name being created.